File: C/lista1 - estrutura de decisao/exercicio_3.c
#include <conio.h>
#include <stdio.h>

int main() {
	int grade;
	
	printf("Informe uma nota: ");
	scanf("%d", &grade);
	
	if (grade < 3) {
		printf("Conceito E.");		
	} else if (grade >= 3 && grade <= 5) {
		printf("Conceito D.");
	} else if (grade >= 6 && grade <= 7) {
		printf("Conceito C.");
	} else if (grade >= 8 && grade <= 9) {
		printf("Conceito B.");
	} else if (grade == 10) {
		printf("Conceito A.");
	}
	
	getch();
	return 0;
}

File: Python/lista1 - estrutura de decisao/exercicio_3.py
grade = float(input('Informe sua nota: '))

if grade < 3:
    print('Conceito E.')
elif grade >= 3 and grade <= 5:
    print('Conceito D.')
elif grade >= 6 and grade <= 7:
    print('Conceito C.')
elif grade >= 8 and grade <= 9:
    print('Conceito B.')
elif grade == 10:
    print('Conceito A.')